grails-app/services/au/org/ala/userdetails/EmailService.groovy

```diff
@@ -42,19 +42,31 @@ class EmailService {
           to user.email
           body (view: '/email/activateAccount',
                 plugin:"email-confirmation",
-                model:[link:getServerUrl() + "activateAccount/" + user.id + "/"  + authKey, orgNameLong: grailsApplication.config.skin.orgNameLong ]
+                model:[userName: user.firstName, link: getServerUrl() + "activateAccount/" + user.id + "/"  + authKey, orgNameLong: grailsApplication.config.skin.orgNameLong ]
           )
         }
     }
 
-    def sendAccountActivationSuccess(def user, def activatedAlerts) {
+    def sendAccountActivationSuccess(user, activatedAlerts) {
         sendMail {
             from grailsApplication.config.emailSenderTitle+"<" + grailsApplication.config.emailSender + ">"
             subject "Account activated successfully"
             to user.email
             body (view: '/email/activateAccountSuccess',
                     plugin:"email-confirmation",
-                    model:[activatedAlerts: activatedAlerts, alertsUrl: grailsApplication.config.alerts.url]
+                    model:[userName: user.firstName, activatedAlerts: activatedAlerts, alertsUrl: grailsApplication.config.alerts.url]
+            )
+        }
+    }
+
+    def sendUpdateProfileSuccess(user) {
+        sendMail {
```

Comment:
Sending mail is a slightly error-prone activity, so I'd be inclined to add exception handling. In particular servers can be deployed with missing or incorrect mail settings, so worthwhile catching this early - and giving user a meaningful error message (as opposed to generic 500 page that they would likely get with implicit exception handling from Grails controllers). See https://github.com/AtlasOfLivingAustralia/userdetails/blob/dev/grails-app/services/au/org/ala/userdetails/EmailService.groovy#L23

Reply:
can do